services/src/test/scala/cromwell/pipeline/service/ProjectServiceTestImpl.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.pipeline.service
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.model.wrapper.UserId
+
+import scala.concurrent.Future
+
+class ProjectServiceTestImpl(project: Project, testMode: TestMode) extends ProjectService {
+
+  private[service] def getUserProjectById(projectId: ProjectId, userId: UserId): Future[Project] =
```

Comment:
We shouldn't need it anymore

Reply:
It is used in several service classes tests and belongs to `ProjectService` trait. Should I `???` it?

Reply:
Got it. Then leave it as is